### PR TITLE
Well-based + Countable pseudocharacter ==> First countable.

### PR DESCRIPTION
**Pull request by Almanzoris**
Note: change last "first countable" to {P28}.

### CODE REVIEW

**Review by StevenClontz — COMMENTED** [review]
This is a bit long for a result without any references. Any chance it's been discussed or asserted in a textbook, or at least a MathSE or MathOverflow conversation?

File: theorems/T000518.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  P000028: true
+---
+
+Let $x \in X$. As $X$ is {P174}, then $x$ has a neighborhood base, let's denote as $\mathcal{B}$, totally ordered by reverse inclusion. And, as $X$ is {P191}, then $\exists \{A_n\}_{n \in \mathbb{N}}$ collection of open sets such that $\bigcap_{n \in \mathbb{N}}A_n = \{x\}$.
+
+For each, $n \in \mathbb{N}$, $\exists B_n \in \mathcal{B}$ such that $B_n \subseteq A_n$. $\{B_n\}_{n \in \mathbb{N}}$ is countable, totally ordered by reverse inclusion and it verifies that $\bigcap_{n \in \mathbb{N}}B_n = \{x\}$. 
+
+Let A be a neighborhood of $x$. Then $\exists B \in \mathcal{B}$ such that $B \subseteq A$. Suppose that $\nexists B_n \in \{B_n\}_{n \in \mathbb{N}}$ such that $B_n \subseteq B$. Then, $B \subsetneq B_n, \forall n \in \mathbb{N}$, and $B \subseteq \bigcap_{n \in \mathbb{N}}B_n = \{x\}$. As a result, we conclude that $B = \{x\}$.
+
+Therefore, $X$ is first countable.
```
Comment (StevenClontz) [inline theorems/T000518.md:17]:
```suggestion
Therefore, $X$ is {P28}.
```

**Comment by Almanzoris** [comment]
> This is a bit long for a result without any references. Any chance it's been discussed or asserted in a textbook, or at least a MathSE or MathOverflow conversation?

Hmmm, I don't know. I came up with it. I can look for it on MSE and MO later.

**Comment by StevenClontz** [comment]
Also compare with https://topology.pi-base.org/theorems/T000217 and note https://topology.pi-base.org/spaces?q=%7Ecountably+tight+%2B+Has+points+%24G_%5Cdelta%24

**Comment by Almanzoris** [comment]
> Also compare with https://topology.pi-base.org/theorems/T000217 and note https://topology.pi-base.org/spaces?q=%7Ecountably+tight+%2B+Has+points+%24G_%5Cdelta%24

Yeah, I have seen Patrick's question on MSE. I hadn't checked this yet, but now that you are showing it, it's significant thing to bear in mind. I haven't found any proofs or counterexamples yet. However, I have found this question: https://mathoverflow.net/q/266331.

**Comment by StevenClontz** [comment]
(No worries about edits!)

**Comment by prabau** [comment]
Yeah, it would be interesting to see the relationship between countably tight and countable pseudocharacter (= points are $G_\delta$)

S154 (Fort space on an uncountable set) is countably tight, but points are not $G_\delta$.

How about the reverse: Is there an example of [points $G_\delta$] and not countably tight?
S131 (Sequential fan with ω-many spines) has both properties.  I thought about a Sequential fan with uncountably many spines, but it seems it also has both properties.
Would be worth asking for an example on mathse.

**Comment by StevenClontz** [comment]
https://math.stackexchange.com/questions/4962337/

**Comment by prabau** [comment]
In the mathse question, can you also mention "countable pseudocharacter" somewhere, as many people know the first property by that name?
(fyi, https://math.meta.stackexchange.com/questions/37668/why-was-this-question-closed-too-niche about a previous mathse topology question I had asked that was closed because deemed too terse ...)

**Comment by prabau** [comment]
Not sure if the proposed T518 can be improved later or not, but it is a nice result that we should add to pi-base.  I agree with Steven that the proof is a little long and it would be preferable to ask it on mathse and refer to that.  

@Almanzoris, how would you like to proceed?  You can ask a question there and answer yourself, or for example I can ask the question and you answer it there, or something else?

(A small request.  Please do not use quantifier symbols in the explanations.  It's clearer to use English.)

**Comment by Almanzoris** [comment]
> Not sure if the proposed T518 can be improved later or not, but it is a nice result that we should add to pi-base. I agree with Steven that the proof is a little long and it would be preferable to ask it on mathse and refer to that.
> 
> @Almanzoris, how would you like to proceed? You can ask a question there and answer yourself, or for example I can ask the question and you answer it there, or something else?
> 
> (A small request. Please do not use quantifier symbols in the explanations. It's clearer to use English.)

I think that you could ask the question and I would answer it. I will avoid quantifier symbols.

**Comment by prabau** [comment]
> @prabau. I think that you could ask the question and I would answer it. I will avoid using quantifier symbols.

Ok.  I'll ask the question now and post the link here when it's available.

**Comment by prabau** [comment]
https://math.stackexchange.com/questions/4963514/is-every-well-based-space-with-countable-pseudocharacter-first-countable

**Comment by Almanzoris** [comment]
I have just come up with a new one, submetrizable implies countable pseudocharacter. The topology contains a coarser topology which is metrizable, so which has countable pseudocharacter. In other words, the topology refines a topology that has points $G_{\delta}$.

**Comment by StevenClontz** [comment]
Sounds like a separate PR to me, but I can confirm it answers an open search: https://topology.pi-base.org/spaces?q=Submetrizable%2B%7EHas+points+%24G_%5Cdelta%24

**Comment by Almanzoris** [comment]
Alright, I will make it after this one.

**Comment by prabau** [comment]
One minor thing.  When we use a mathse/mo post as a reference, we use a reference to a specific answer (with `name: Answer to "..."`) only if it is not clear that the question itself is about the relevant topic and we need to refer to a specific answer.  But if the question itself is about the correct topic and we don't need to specify a specific answer, it's preferable to refer to the whole post (without "Answer to" in the `name:`.  In this case, you can just refer to the id 4963514.

**Comment by prabau** [comment]
Looks good to me.  @StevenClontz I leave it to you to approve.

**Review by StevenClontz — APPROVED** [review]

**Pull request merged**